examples/colab/tf2_hub_single_image_super_resolution_ESRGAN.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>why can't you calculate psnr for smaller images?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Well, the if the input shape is less than the crop size, then the smaller dimension will remain unchanged producing different High resolution and low resolution images.</p><p>However PSNR can be calculated between images of same shape.</p>

Reply:
<p>So then do you have to crop at all? What about just leaving the cropping out and working with any kind of image on input?</p>

Reply:
<p>Yea, that's true. since we are accepting all images. Okay!</p>